clang/test/SemaSYCL/sycl-fptr-lambda.cpp

```diff
@@ -0,0 +1,16 @@
+// RUN: %clang_cc1 -fsycl-is-device -std=c++14 -verify -fsyntax-only -x c++ %s
+// expected-no-diagnostics
+
+template <typename name, typename Func>
+__attribute__((sycl_kernel)) void kernel_single_task(Func kernelFunc) {
+  kernelFunc();
+}
+
+int main() {
+  kernel_single_task<class fake_kernel>([]() {
+		                   // expected-no-error@+1
```

Comment:
We already have `// expected-no-diagnostics` at the beginning of the file, so this check seems redundant. Right?

Reply:
Yes, it is.  Removed.  Thanks.